### PR TITLE
Updated the FAQ

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR updates the FAQ according to received feedback.

### CODE REVIEW

File: pages/faqs.md [hunk 1]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+
+In Europe <a href="https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32016R0679&from=EN#d1e1489-1-1">personal data is defined under the General Data Protection Regulation (GDPR)</a> as "any information relating to an identified or identifiable natural person (‘data subject’); an identifiable natural person is one who can be identified, directly or indirectly, in particular by reference to an identifier such as a name, an identification number, location data, an online identifier or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural or social identity of that natural person;"
+</details>
+
+<a id="pod"/>
 
 ## What is a Pod? 
-In Solid, a Pod is where a user stores their data. 
+
```
Comment (michielbdejong) [inline pages/faqs.md]: "where users store" (plural)

Comment (michielbdejong) [reply pages/faqs.md]: "store on the web" may sound like public, so maybe say "where users securely store" or use the phrase "personal data vault", I always feel that works well with quite a wide audience.

Also, just noting you're mixing pod singular with users plural in this sentence, although I don't think it's necessarily problematic in this case.

File: pages/faqs.md [hunk 2]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+<a id="webid"/>
 
 ## What is a WebID? 
-In Solid, a WebID is the way you login and connect to your contacts. 
 
-## To what extent are Pods conceived as distinct from the WebIDs?
+**A WebID is a unique identifier** used to identify a specific user. An example of what a WebID could look like is: https://fulano.pod.provider/profile/card#me. To share data with a third party, a user associates sharing preferences to the WebID of that third party. Don't worry though: [it's not just one more login to remember](#passwords). 
+<details>
+<summary><a>More details...</a></summary>
+A WebID is an <a href="https://tools.ietf.org/html/rfc3987">Internationalised Resource Identifier (IRI)</a> that can be dereferenced as a profile document serialized in <a href="https://www.w3.org/RDF/">RDF</a> <a href="https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html">source</a>. In Solid, WebIDs are used to identify Agents i.e. people and organisations as well as to manage their access rights though <a href="https://solid.github.io/specification/wac/">Web Access Control</a>.
```
Comment (michielbdejong) [inline pages/faqs.md]: Put "source" in brackets, as :"(source)"

File: pages/faqs.md [hunk 3]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+<details>
+<summary><a>More details...</a></summary>
+A WebID is an <a href="https://tools.ietf.org/html/rfc3987">Internationalised Resource Identifier (IRI)</a> that can be dereferenced as a profile document serialized in <a href="https://www.w3.org/RDF/">RDF</a> <a href="https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html">source</a>. In Solid, WebIDs are used to identify Agents i.e. people and organisations as well as to manage their access rights though <a href="https://solid.github.io/specification/wac/">Web Access Control</a>. 
+</details>
+
+## How are Pods different from WebIDs?
 
-An identity provider delivers your WebID as a form of authentication. The identity provider is the domain your WebID Profile is found. A Pod provider delivers one or more domains pointed to by solid:storage statements which can be the same of different to your identity provider.
+**The two are different things**: [the WebID is an identifier](#webid), and [the Pod is a data storage place](#pod). They are provided by two distinct entities:
```
Comment (michielbdejong) [inline pages/faqs.md]: two potentially distinct

**Comment by NSeydoux** [comment]
Just FYI (especially for @Mitzi-Laszlo), i updated the "How can I delete my WebID" question in order to make it more general, in the idea of "If I want to leave Solid, how to do it ?", this way it's also about deleting Pods.

File: pages/faqs.md [hunk 4]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+<summary><a>More details...</a></summary>
+A WebID is an <a href="https://tools.ietf.org/html/rfc3987">Internationalised Resource Identifier (IRI)</a> that can be dereferenced as a profile document serialized in <a href="https://www.w3.org/RDF/">RDF</a> <a href="https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html">source</a>. In Solid, WebIDs are used to identify Agents i.e. people and organisations as well as to manage their access rights though <a href="https://solid.github.io/specification/wac/">Web Access Control</a>. 
+</details>
+
+## How are Pods different from WebIDs?
 
-An identity provider delivers your WebID as a form of authentication. The identity provider is the domain your WebID Profile is found. A Pod provider delivers one or more domains pointed to by solid:storage statements which can be the same of different to your identity provider.
+**The two are different things**: [the WebID is an identifier](#webid), and [the Pod is a data storage place](#pod). They are provided by two distinct entities: 
+- The Pod Provider hosts Pods
+- The Identity Provider hosts WebIDs.
```
Comment (michielbdejong) [inline pages/faqs.md]: Hm, well, the hosting of the profile document is often still done by the storage provider, the IDP only hosts authentication services that allow you to prove that you own the WebID.

File: pages/faqs.md [hunk 5]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+</details>
+
+## How are Pods different from WebIDs?
 
-An identity provider delivers your WebID as a form of authentication. The identity provider is the domain your WebID Profile is found. A Pod provider delivers one or more domains pointed to by solid:storage statements which can be the same of different to your identity provider.
+**The two are different things**: [the WebID is an identifier](#webid), and [the Pod is a data storage place](#pod). They are provided by two distinct entities: 
+- The Pod Provider hosts Pods
+- The Identity Provider hosts WebIDs. 
+
+To log into Solid apps and/or access data on Pods, you just have to use your WebID to login to you Identity Provider, like "Sign in with Google" today. It is possible for the same company or organisation to be both a Pod Provider and a Identity Provider although they are **distinct seperable services** that are compatible with other Pod Providers and Identity Providers (which means you are free to choose the provider that suits you the most).
```
Comment (michielbdejong) [inline pages/faqs.md]: Hm, you don't use your WebID to log in to your IDP unless you use the WebID-TLS bridge. Maybe change it to:

you just have to *provide* your WebID *and* login to you Identity Provider

Comment (michielbdejong) [reply pages/faqs.md]: s/seperable/separable

File: pages/faqs.md [hunk 6]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
 
-An identity provider delivers your WebID as a form of authentication. The identity provider is the domain your WebID Profile is found. A Pod provider delivers one or more domains pointed to by solid:storage statements which can be the same of different to your identity provider.
+**The two are different things**: [the WebID is an identifier](#webid), and [the Pod is a data storage place](#pod). They are provided by two distinct entities: 
+- The Pod Provider hosts Pods
+- The Identity Provider hosts WebIDs. 
+
+To log into Solid apps and/or access data on Pods, you just have to use your WebID to login to you Identity Provider, like "Sign in with Google" today. It is possible for the same company or organisation to be both a Pod Provider and a Identity Provider although they are **distinct seperable services** that are compatible with other Pod Providers and Identity Providers (which means you are free to choose the provider that suits you the most). 
+<details> 
+<summary><a>More details...</a></summary>
+<p>An Identity Provider owns the domain your WebID Profile is found, and implements an identification protocol (e.g. [OIDC](https://openid.net/connect/)).</p>
```
Comment (michielbdejong) [inline pages/faqs.md]: the domain your WebID Profile is found
the domain your WebID Profile is at

File: pages/faqs.md (hunk 4 above)
Comment (NSeydoux) [reply pages/faqs.md]: Correct, and I think this has implications in multiple places (e.g., the deletion of an account). I'll make sure that the whole page is consistent on that.

File: pages/faqs.md (hunk 6 above)
Comment (michielbdejong) [reply pages/faqs.md]: OK, if we define it that way, then the IDP *does* host your WebID Profile. OK, that's fine, and probably less confusing that way.

File: pages/faqs.md [hunk 7]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
 
 ## How does Solid relate to other Web standards?
-Solid defines how apps and Pods communicate data by defining a single API. The client-server API is made up of [HTTP]( https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol), [LDP]( https://en.wikipedia.org/wiki/Label_Distribution_Protocol), and [WAC]( https://www.w3.org/wiki/WebAccessControl). 
 
-Users have more options of where to store the data generated while using an app as well as who has access to what data, when and under which conditions. 
+**Solid does not reinvent the wheel**. When you use Solid you will still be able to access your data using the same Web browser on the same computer. Solid is still the web, but with a few things added and a few assumptions overturned ((see quote at 33:30](https://www.bbc.co.uk/programmes/m000bj15)). 
+
+Solid adds: 
+* An identification system so that instead of having to sign in to everything with Google or Facebook you can sign in with your favourite Solid provider and you won't be tracked. 
+* Ubiquitous sharing control so that you can share anything with anybody no matter which social media they happen to be part of. Solid
```
Comment (michielbdejong) [inline pages/faqs.md]: Well, that's maybe not strictly true. You can publish content on your pod and then log in to Facebook to reshare it there. Other than that I don't think it's very integrated.

Remove " Solid" at the end of the line.

File: pages/faqs.md [hunk 8]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+Solid overturns assumptions: 
+* When you start a Solid app, instead of the app storing the data itself, it stores it on your Pod. When you use the app, you choose which app to use, and you choose which Pod to use with which app. 
+<details>
+<summary><a>More details</a></summary>
+Solid is built on top of existing Web and Linked Data standards as much as possible. The core Solid specification relies on <a href="https://www.w3.org/TR/ldp/">LDP</a> and <a href="https://solid.github.io/specification/wac/">WAC</a> (<a href="https://github.com/solid/web-access-control-spec">WAC draft</a>, both being based on <a href="https://tools.ietf.org/html/rfc2616">HTTP</a> and <a href="https://www.w3.org/RDF/">RDF</a> vocabularies. Solid also uses <a href="https://www.w3.org/TR/sparql11-overview/">SPARQL</a> through HTTP PATCH queries. Identification in Solid is based on <a href="https://www.w3.org/2005/Incubator/webid/spec/tls/">WebID-TLS</a> and/or <a href="https://openid.net/connect/">OIDC</a>.
```
Comment (michielbdejong) [inline pages/faqs.md]: It uses a subset of sparql-update  for PATCH, and currently still a subset of SPARQL itself on GET, but the latter will probably disappear in the 19 December version of the spec. So it's not really using SPARQL.

Comment (michielbdejong) [reply pages/faqs.md]: Identification is based on WebID-OIDC.

File: pages/faqs.md [hunk 9]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
 # Storage 
 
 ## Are Solid users expected to setup their own servers and self-host?
 
-You can choose to self-host or select a hosting provider. Self-hosting means that your data would sit at home on your own physical hard drive.
+**No**. Self-hosting means that your data would sit at home on your own physical hard drive or server. Self-hosting is possible but not essential when using Solid, and it currently requires some technical knowledge. A Solid user can rely on an Identity Provider and a Pod Provider, and is **not expected to have any particular expertise**. It is also possible to self-host and become a small Identity Provider and Pod Provider, to provide Solid to your family, association, friends...
+
+<details>
+<summary><a>More details</a></summary>
+The Solid standard is open as are some of its implementations such as such as [Node Solid Server](https://github.com/solid/node-solid-server) meaning that anyone can self-host their own Identity and Pod or the Identity and Pod of a group of users.
```
Comment (michielbdejong) [inline pages/faqs.md]: such as such as

File: pages/faqs.md [hunk 10]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
 
 ## Is it possible to use Solid offline (at least partially)? 
 
-Yes, the Solid long term vision includes local first and a flexibility of different topologies of patch-passing sync networks. However, there are no implementations yet. 
+**Eventually, yes**. The Solid long term vision includes local first and a flexibility of different topologies of patch-passing sync networks. However, there are no implementations yet. 
 
 ## When I want to leave a Pod provider, can I take my data with me?
-This depends on the agreement you have with your hosting provider. There are some support tools like [Terms of Service Don't Read](https://tosdr.org) to help you understand the small print more easily.
+Yes, unless the Pod provider doesn't allow you to. The best way to understand the service of a particular Pod provider is to read the Terms. There are some support tools like [Terms of Service Don't Read](https://tosdr.org) to help you understand the small print more easily.
```
Comment (michielbdejong) [inline pages/faqs.md]: Hey, thanks for the ToS;DR shout-out ;)

File: pages/faqs.md [hunk 11]
```diff
@@ -9,69 +9,125 @@ Got a question that's not here? Try asking on the [Solid forum](https://forum.so
...
+The data in Solid Pods is structured according to Linked Data principles by the applications writing them in the first place. Therefore, the Pod Provider is completely neutral regarding the Pod <b>content</b>, and the same data structure should be supported by any Pod Provider. The storage technology picked by each Pod Provider is <a href="#how-storage">another question</a>. 
+</details>
+
+<a id="how-storage"/>
 
-## When using Solid, how is data stored?
-Solid provides a general purpose read-write REST API to store data in individual files (resources) and directories (containers).
+##  When using Solid, how is data stored? 
+
+**It depends on the Pod Provider**. From a user point of view, how the data is stored is not as important as how it is accessed and controlled. No matter who the Pod Provider is, in order to be Solid compliant, it has to expose data the same way: as resources in folders. However,the implementors of the standard are free to pick the underlying technologies according to their own purposes and constraints. That is why performances may vary from one Pod Provider to another.
```
Comment (michielbdejong) [inline pages/faqs.md]: performance may

File: pages/faqs.md [hunk 12]
```diff
@@ -80,40 +136,58 @@ It’s a great start. The next step is to build a healthy array of options for u
 
 ## How to integrate iOS and Android apps into Solid? 
 
-Currently there are no developer kits in development for Android or iOS.If you really want to have an app that can be installed on Android or iOS, you might consider writing it as a progressive web app or writing it as a hybrid app. 
+It is possible for developers to take an iOS or Android app and make it Solid compatible by following the standard and supporting documentation. 
+
+<details>
+<summary><a>More details</a></summary>
+Currently there are no developer kits in development for Android or iOS. If you really want to have an app that can be installed on Android or iOS, you might consider writing it as a progressive web app or writing it as a hybrid app.
```
Comment (michielbdejong) [inline pages/faqs.md]: We can say that people (namely, @jaxoncreed) are working on a rewrite of Solid-Auth-Client which will work with React-Native, right?

File: pages/faqs.md [hunk 13]
```diff
@@ -80,40 +136,58 @@ It’s a great start. The next step is to build a healthy array of options for u
...
+</details>
 
 ## Arguably the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web) or [Linked Data](https://en.wikipedia.org/wiki/Linked_data) never took off, so why is Solid working with it? 
 
-Today, almost a quarter of all websites are currently using the semantic web or linked data. The Solid community is working to provide seamless convenience to developers to work with the semantic web or linked data more easily.
+The Semantic Web and Linked Data principles and technologies have always meant to **make data more reusable**, and to **make data independant from applications**, which is why it is at the core of Solid.
```
Comment (michielbdejong) [inline pages/faqs.md]: s/independant/independent

File: pages/faqs.md [hunk 14]
```diff
@@ -80,40 +136,58 @@ It’s a great start. The next step is to build a healthy array of options for u
...
 
-## How will Solid prevent 3rd parties from replicating data?
-Solid currently is working on access controls to selectively limit the data access to applications. 
+<a id="copy-after-revoking"/>
 
-## Can I make sure that nobody has a copy of my data? 
-Solid does not get the data you already sent out back. As soon as you start using Solid, you will be able to be more deliberate about your data sharing preferences from that point onwards. If you would like to ask for your data to be deleted from other services, In Europe there is a law called the [Right to be Forgotten](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32016R0679&from=EN#d1e2589-1-1) which makes it possible for European citizens to ask for their data to be deleted. You can find template letters and instructions on how to exercise this right [here](https://www.datarequests.org/blog/sample-letter-gdpr-erasure-request/). 
+## When I revoke access of an application to my data, can they still hold a copy of the data they accessed?
```
Comment (michielbdejong) [inline pages/faqs.md]: if you want this question to be able to stand on its own out of context, change 'they' to 'others'

File: pages/faqs.md [hunk 15]
```diff
@@ -122,16 +196,20 @@ The law determines the remit of the government to track or control criminal acti
 Pod providers have commitments to the individuals and legal entities that they provide services to based on the law of the countries in which the hosting provider is operating. As a Pod provider you should seek professional advice on how to be compliant. As an individual Solid user, you should carefully read the agreement with your hosting provider before agreeing. 
 
 ## Could Solid become another tech giant monopoly?
-Solid is an open standard. Could the solid standards dominate the technical landscape? Yes, that would be ideal, so we would have one interoperable Solid web, not many.
+
+**No**. Solid is **not a company** it is an open standard. Could it dominate the technical landscape, by being adopted by many? Yes, that would be ideal, so we would have one interoperable Solid web, not many. However, being compliant with Solid does not require developpers to ask permission from anyone before building their applications, and does not prevent users from choosing the apps they want, and share their data how they see fit. In that sense, Solid cannot become a monopoly, in the same way that the Web is not considered a tech monopoly.
```
Comment (michielbdejong) [inline pages/faqs.md]: not a company, it is

**Comment by NSeydoux** [comment]
Actually, I'm not sure about what happens when you want to leave the Solid ecosystem. If your profile document is hosted in your Pod, but your Pod Provider is not your Identity Provider, what should the course of actions be ?

File: pages/faqs.md (hunk 10 above)
Comment (Vinnl) [reply pages/faqs.md]: I actually had that removed at some other place in the website, given that it doesn't have a review of the major Pod providers now was unlikely to get one any time soon :P

**Comment by michielbdejong** [comment]
> If your profile document is hosted in your Pod, but your Pod Provider is not your Identity Provider, what should the course of actions be ?

Sometimes, your pod provider will only identify you by your webId, so you should first delete your pod, and then delete your IDP account.

URLs of deleted accounts should keep a tombstone record, and never be reused, otherwise someone else could impersonate you after you left.

**Pull request merged**